front-end/src/components/newsfeed-view/news-list/news-card/NewsCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Timestamp from './elements/Timestamp';
+import './newscard.css';
+
+
+const NewsCard = ({data}) => (
```

Comment:
seems like NewsCard and InstagramNewsCard isnt that different. Perhaps you can have a single NewsCard component and have slightly diffrent rendering method based on the incoming data. This scales better with more types of card coming, or with varying data structure (which we will most likely have)